src/main/kotlin/org/wa/rceditor/application/view/fragments/ContributorItemFragment.kt

```diff
@@ -30,7 +30,9 @@ class ContributorItemFragment: ListCellFragment<ContributorItem>() {
         }
         button(graphic = Styles.closeIcon()) {
             removeWhen { parent.hoverProperty().not().or(editingProperty) }
-            action { viewModel.removeContributor(item) }
+            action {
+                viewModel.removeContributor(item)
```

Comment:
Does it work to do `items.remove(item)` here? Trying to move away from tying this list cell fragment to a particular field (like contributor) or view model

Reply:
@mbr4477  Nope, list cell fragment doesn't have an access to the items property

Reply:
Ah okay. Maybe there's a way to commit a null item on delete? Then the listview onEdit callback can delete it when a null gets committed? No idea if that would work, just throwing out ideas at this point. I'd say work on the other things first and then maybe come back to this at the end